project/project/CakePlatformDependencies.scala

```diff
@@ -113,7 +113,6 @@ object CakePlatformDependencies {
     val packager: ModuleID =
```

Comment:
We also want to remove a number of the other Play related dependencies here - e.g. webjars grouping, janino(?), playJson, sbt-js-engine and sbt-web.